package.json

```diff
@@ -9,12 +9,12 @@
     "build:date-adapters": "tsc -p tsconfig-date-adapters.json",
     "build:styles": "node-sass projects/angular-calendar/src/angular-calendar.scss | postcss --output dist/angular-calendar/css/angular-calendar.css",
     "build:clean": "del-cli dist",
-    "build:schematics": "tsc projects/angular-calendar/schematics/ng-add/index.ts",
+    "build:schematics": "tsc --skipLibCheck projects/angular-calendar/schematics/ng-add/index.ts",
     "build:copyfiles": "copyfiles CHANGELOG.md README.md LICENSE dist/angular-calendar & cp-cli projects/angular-calendar/util/date-adapter-package.json dist/angular-calendar/date-adapters/date-fns/package.json & cp-cli projects/angular-calendar/util/date-adapter-package.json dist/angular-calendar/date-adapters/moment/package.json & copyfiles -u 3 projects/angular-calendar/src/**/*.scss dist/angular-calendar/scss & npx copyfiles -u 2 \"projects/angular-calendar/schematics/**/*.{json,js}\" dist/angular-calendar",
     "build": "run-s build:copy-package-json build:lib build:date-adapters build:styles build:schematics build:copyfiles",
     "test:single": "cross-env TZ=UTC ng test angular-calendar --watch=false --code-coverage",
     "test:watch": "cross-env TZ=UTC ng test angular-calendar",
-    "test:schematics": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' mocha --require ts-node/register projects/angular-calendar/schematics/**/*.spec.ts",
+    "test:schematics": "cross-env TS_NODE_COMPILER_OPTIONS='{\\\"module\\\":\\\"commonjs\\\"}' mocha --require ts-node/register projects/angular-calendar/schematics/**/*.spec.ts",
```

Comment:
This fails on unix, removing the single quotes should fix it e.g. `TS_NODE_COMPILER_OPTIONS='{\\\"module\\\":\\\"commonjs\\\"}'` -> `TS_NODE_COMPILER_OPTIONS={\\\"module\\\":\\\"commonjs\\\"}`